src/widgets/contact.jsx

```diff
@@ -70,7 +70,10 @@ class Contact extends React.Component {
 
     // The this.props.preview contains alreay shortened Drafty or string.
     let preview;
-    if (typeof this.props.preview == 'string') {
+    if (this.props.previewIsVideoCall) {
+      preview = <><i className='material-icons'>{this.props.previewIsResponse ? 'call_received' : 'call_made'}</i>
```

Comment:
Please use single/double quotes consistently in `className='material-icons'`. The jsx is usually double quotes.